gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const zip = require('gulp-zip');
+
+gulp.task('build', ['lint', 'clean:dist'], function () {
+  return gulp.src('src/*')
+    .pipe(zip('addon.xpi'))
+    .pipe(gulp.dest('dist'));
+});
```

Comment:
> We can do a future PR to copy the *.rdf files into dist/ too if that seems useful.

I really like this idea. Only releasing from `dist` is a nice concept. Feel free to file a bug to do this later, or tack it onto this PR if you're feeling ambitious (if you do it now, update the release steps in the README to pull update.rdf from dist instead of src).